src/js/utils/filter-source.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @module filter-source
+ */
+import {isObject} from './obj';
+
+/**
+ * Filter out single bad source objects or multiple source objects in an
```

Comment:
I will remove that from the doc comments/unit tests if you like, but the code will probably stay the same either way.